server/public/model/cluster_message.go

```diff
@@ -16,7 +16,7 @@ const (
 	ClusterEventInvalidateCacheForChannel                   ClusterEvent = "inv_channel"
 	ClusterEventInvalidateCacheForChannelGuestCount         ClusterEvent = "inv_channel_guest_count"
 	ClusterEventInvalidateCacheForUser                      ClusterEvent = "inv_user"
-	ClusterEventInvalidateCacheForUserTeams                 ClusterEvent = "inv_user_teams"
+	ClusterEventInvalidateWebConnCacheForUser               ClusterEvent = "inv_user_teams"
```

Comment:
I renamed the constant name but kept the actual value to be the same to preserve the Grafana charts.